src/properties/mod.rs

```diff
@@ -525,13 +525,19 @@ pub struct PropertyOptions<'a> {
 
 impl<'a> PropertyOptions<'a> {
     /// Get custom version
-    pub fn get_custom_version<T>(&self) -> FCustomVersion
+    pub fn get_custom_version<T>(&self) -> u32
```

Comment:
I think get custom version should still return `FCustomVersion`, we could allow something like `Into<u32>` for it though

Reply:
My thinking was that since the Guid is essentially the input to this function, it is unnecessary to return that value back to the caller. I decided to add `#[inline]` so that the change you requested should not have any impact on performance.